.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
No! I remembered! There was a reason: i used **Symfony\Component\DependencyInjection\ContainerAwareTrait** in controllers because in Symfony3 **Symfony\Component\DependencyInjection\ContainerAware** was removed and there was no traits in PHP 5.3 and PHP 5.4